docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12.5-alpine
+
+ARG GOOSEBIT_VERSION
+
+RUN addgroup -g 1000 goosebit && \
+    adduser -u 1000 -G goosebit -s /bin/sh -D goosebit && \
+    pip install --no-cache-dir \
+        gunicorn \
+        goosebit[postgresql]==$GOOSEBIT_VERSION
+
+COPY aerich.toml /
+
+VOLUME /artifacts
+
+EXPOSE 60053
+
+USER goosebit
+
+ENV GUNICORN_CMD_ARGS="--workers 2 --enable-stdio-inheritance"
+
+CMD aerich --config /aerich.toml upgrade && \
```

Comment:
This (in conjunction with #108) doesn't seem to be enough for an empty PostgreSQL DB. I get the following:
```
postgres-1  | 2024-08-30 06:43:01.705 UTC [76] ERROR:  relation "aerich" does not exist at character 44
postgres-1  | 2024-08-30 06:43:01.705 UTC [76] STATEMENT:  SELECT "id","content","app","version" FROM "aerich" WHERE "app"='models' ORDER BY "id" DESC LIMIT 1
postgres-1  | 2024-08-30 06:43:01.706 UTC [76] ERROR:  relation "aerich" does not exist at character 15
postgres-1  | 2024-08-30 06:43:01.706 UTC [76] STATEMENT:  SELECT 1 FROM "aerich" WHERE "version"='0_20240828095320_init.py' AND "app"='models' LIMIT 1
goosebit-1  | Traceback (most recent call last):
goosebit-1  |   File "/usr/local/bin/aerich", line 8, in <module>
goosebit-1  |     sys.exit(main())
goosebit-1  |              ^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/aerich/cli.py", line 265, in main
goosebit-1  |     cli()
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
goosebit-1  |     return self.main(*args, **kwargs)
goosebit-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1078, in main
goosebit-1  |     rv = self.invoke(ctx)
goosebit-1  |          ^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
goosebit-1  |     return _process_result(sub_ctx.command.invoke(sub_ctx))
goosebit-1  |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
goosebit-1  |     return ctx.invoke(self.callback, **ctx.params)
goosebit-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 783, in invoke
goosebit-1  |     return __callback(*args, **kwargs)
goosebit-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/click/decorators.py", line 33, in new_func
goosebit-1  |     return f(get_current_context(), *args, **kwargs)
goosebit-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/aerich/cli.py", line 31, in wrapper
goosebit-1  |     loop.run_until_complete(f(*args, **kwargs))
goosebit-1  |   File "/usr/local/lib/python3.12/asyncio/base_events.py", line 687, in run_until_complete
goosebit-1  |     return future.result()
goosebit-1  |            ^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/aerich/cli.py", line 104, in upgrade
goosebit-1  |     migrated = await command.upgrade(run_in_transaction=in_transaction)
goosebit-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/aerich/__init__.py", line 61, in upgrade
goosebit-1  |     await self._upgrade(conn, version_file)
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/aerich/__init__.py", line 43, in _upgrade
goosebit-1  |     await conn.execute_script(await upgrade(conn))
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/tortoise/backends/base_postgres/client.py", line 35, in _translate_exceptions
goosebit-1  |     return await self._translate_exceptions(func, *args, **kwargs)
goosebit-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/tortoise/backends/asyncpg/client.py", line 87, in _translate_exceptions
goosebit-1  |     return await func(self, *args, **kwargs)
goosebit-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/tortoise/backends/base_postgres/client.py", line 153, in execute_script
goosebit-1  |     await connection.execute(query)
goosebit-1  |   File "/usr/local/lib/python3.12/site-packages/asyncpg/connection.py", line 350, in execute
goosebit-1  |     result = await self._protocol.query(query, timeout)
goosebit-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
goosebit-1  |   File "asyncpg/protocol/protocol.pyx", line 374, in query
goosebit-1  |   File "asyncpg/protocol/protocol.pyx", line 887, in asyncpg.protocol.protocol.BaseProtocol._dispatch_result
goosebit-1  |   File "asyncpg/protocol/protocol.pyx", line 814, in asyncpg.protocol.protocol.BaseProtocol._on_result__simple_query
goosebit-1  | AttributeError: 'NoneType' object has no attribute 'decode'
```

Reply:
Turns out the migration was the problem. So, all good.